resources/views/account/register.blade.php

```diff
@@ -11,10 +11,8 @@
                 @lang('Register')
             </x-rapidez-ct::title>
 
-            <x-rapidez-ct::sections>
-                @include('rapidez-ct::account.partials.register-account')
-                @include('rapidez-ct::components.newsletter')
-            </x-rapidez-ct::sections>
+            
+            @include('rapidez-ct::account.partials.register-account')
```

Comment:
@jordythevulder @JimmyHoenderdaal

I've moved the `sections` component to be inside the register-account partial here. Do you think this change is problematic? The issue with the previous approach is that the graphql-mutation component needs to be around both of these things, and Vue won't let you do that the easy way.

Reply:
Would be problematic for extending this template with extra includes since it won't be inside the sections anymore

Reply:
section component is for the spacing between the sections and spacing top and spacing bottom, so i would like to have that component within the sections component. What was the reason for that?

Reply:
I need to have the graphql-mutation vue component outside of the sections because of the following limitations:

- The component needs to encompass both the register and the newsletter
- A component can only have one root node, so it can't be placed inside of the sections component without also breaking the spacings of the sections component

I could also move the graphql-mutation inside of this file, but it's a pretty substantial piece of code:

```blade
<x-rapidez::recaptcha location="customer_create"/>
<graphql-mutation
    query="mutation customer ($firstname: String!, $lastname: String!, $email: String!, $password: String, $is_subscribed: Boolean) { createCustomerV2 ( input: { firstname: $firstname, lastname: $lastname, email: $email, password: $password, is_subscribed: $is_subscribed } ) { customer { email } } }"
    redirect="{{ route('account.overview') }}"
    :callback="registerCallback"
    :recaptcha="{{ Rapidez::config('recaptcha_frontend/type_for/customer_create') == 'recaptcha_v3' ? 'true' : 'false' }}"
    v-slot="{ mutate, variables }"
>
    <x-rapidez-ct::sections>
[...]
```

So, to keep this specific component clean, my decision was to move everything into that register-account component instead.

Reply:
@jordythevulder and @JimmyHoenderdaal, ok with this? Merge?

Reply:
👍